### PR TITLE
First batch of massive update

### PR DESCRIPTION
**Pull request by rufotheone-fr**

### CODE REVIEW

File: compatibility.html [hunk 1]
```diff
@@ -64,11 +64,8 @@
...
-								<p><i class="fas fa-question-circle unknown"></i><span style="font-weight: bold;">Unknown</span>: Compatibility not yet reported.</p>
+								<p><i class="fas fa-check-circle perfect"></i><span style="font-weight: bold;">Perfect</span>: Game is fully emulated and all content can be played.<br /><i class="fas fa-check-circle runs"></i><span style="font-weight: bold;">OK</span>: Game runs and is almost completely playable, but may have some minor issues or missing some minor components.<br /><i class="fas fa-exclamation-circle loads"></i><span style="font-weight: bold;">Issues</span>: Game runs and can play most content, but some components may not be emulated fully.<br /><i class="fas fa-minus-circle unplayable"></i><span style="font-weight: bold;">Unplayable</span>: Game loads but may have some significant components not fully emulated.<br /><i class="fab fa-patreon"></i><span style="font-weight: bold;">Patreon</span>: Game access limited to Patreons.<br /><i class="fas fa-question-circle unknown"></i><span style="font-weight: bold;">Unknown</span>: Compatibility not yet reported.</p>
```
Comment (Poliwrath) [inline compatibility.html]: Why?

Comment (rufotheone-fr) [reply compatibility.html]: Why what?

Comment (Poliwrath) [reply compatibility.html]: I don't believe it's necessary... maybe @sTo0z can chime in

Comment (rufotheone-fr) [reply compatibility.html]: It is. A line break <br /> takes less space than a paragraph <p>

Comment (Poliwrath) [reply compatibility.html]: Yes, but you're sacrificing readability for "compactness". There should be no reason to do this...

Comment (rufotheone-fr) [reply compatibility.html]: Readability of raw code or rendered htm?
Because I think that you don't know what the result looks like.

Comment (Poliwrath) [reply compatibility.html]: Yes, html code
Having one very long line is annoying...

Comment (rufotheone-fr) [reply compatibility.html]: If it's just that, I can fix this.

Comment (Poliwrath) [reply compatibility.html]: Perfect. Now I think you just need to wait for Sto0z or Reaver to check it out

Comment (sTo0z) [reply compatibility.html]: What you've done is changed a bunch of block elements into a single element. In this particular case either has its advantages and disadvantages. It does technically collect a bunch of similar elements into a single block, but loses a tiny bit of flexibility if needed in the future. Given the small amount of code either way, this change could go either way. Not a big deal to let it through for now.

**Comment by Poliwrath** [comment]
too many conflicts, don't know if this is still needed...

**Pull request closed without merging**